.github/workflows/docker.yml

```diff
@@ -180,6 +180,10 @@ jobs:
     runs-on: ubuntu-20.04
     needs:
       - build
+    strategy:
+      fail-fast: false
+      matrix:
+        browsers: [chrome, firefox, edge]
```

Comment:
I think we should run it in electron too. Since in practice that surfaced some potential bugs already.

Reply:
Resolved via 6c2ffac.